src/theme/variables/edgeLight.js

```diff
@@ -177,6 +177,11 @@ export const edgeLight: Theme = {
   settingsRowHeaderBackground: palette.lightGray,
   settingsRowSubHeader: palette.transparent,
 
+  // Loader Color
+  walletLoaderFill: palette.edgeMint,
```

Comment:
Change names to start with walletProgressIcon...

Reply:
Fixed